acceptable/util.py

```diff
@@ -1,9 +1,10 @@
 import inspect
+import sys
 
 
 def get_callsite_location(depth=1):
-    frame = inspect.stack()[depth + 1]
+    frame = sys._getframe(depth + 1)
```

Comment:
"CPython implementation detail: This function should be used for internal and specialized purposes only. It is not guaranteed to exist in all implementations of Python.". Is it worth doing a try/except and falling back to slow inspect.stack?

Reply:
pypy supports sys._getframe, are there any other python implementations we care about?

Reply:
I don't know if we care about others :) If you don't, then let's go with this. Worst case, it will explode for someone using IronPython or some such horror and we'll fix a bug at that point.